package.json

```diff
@@ -44,6 +44,8 @@
   "dependencies": {
     "@rollup/plugin-typescript": "^5.0.2",
     "@tsconfig/svelte": "^1.0.10",
+    "install": "^0.13.0",
```

Comment:
Is there a reason this module got... [ahem]... installed? https://www.npmjs.com/package/install

Or was it added by mistake?